src/BoundaryConditions/perturbation_advection_open_boundary_matching_scheme.jl

```diff
@@ -114,7 +114,7 @@ end
 
     τ̃ = Δt / τ
 
-    u₁ⁿ⁺¹ = (uᵢⁿ - U * uᵢ₋₁ⁿ⁺¹ + ūⁿ⁺¹ * τ̃) / (1 + τ̃ - U)
+    u₁ⁿ⁺¹ = isinf(τ̃) ? ūⁿ⁺¹ : (uᵢⁿ - U * uᵢ₋₁ⁿ⁺¹ + ūⁿ⁺¹ * τ̃) / (1 + τ̃ - U)
```

Comment:
can you organize this code into "paragraphs" (we don't want to use double space code I feel)

Reply:
Is this what you're suggesting?

```suggestion
   if isinf(τ̃)
        u₁ⁿ⁺¹ =  ūⁿ⁺¹
    else
        u₁ⁿ⁺¹ =  (uᵢⁿ - U * uᵢ₋₁ⁿ⁺¹ + ūⁿ⁺¹ * τ̃) / (1 + τ̃ - U)
    end
```

Reply:
no no I just mean removing some vertical white space

Reply:
Ah I see. Yeah I agree with you. I also find it harder to read. I'll improve that.

Reply:
Done. Can you check?